src/multibody/fcl.hpp

```diff
@@ -115,24 +115,32 @@ enum GeometryType
   COLLISION
 };
 
-struct GeometryObject
+struct GeometryObject; //fwd
+
+template<>
+struct traits<GeometryObject>
+{
+  typedef double Scalar;
+  enum { Options = 0};
+};
+
+struct GeometryObject : public ModelItem<GeometryObject>
 {
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+  typedef GeometryObject ModelItemDerived;
```

Comment:
It would be nice to not add typedef for nothing.

Reply:
DOne